lib/deprecations.js

```diff
@@ -3,8 +3,6 @@
 const warning = require('fastify-warning')()
 module.exports = warning
 
-const warnName = 'PinoWarning'
```

Comment:
is this file required anywhere else now? if not, remove?

Reply:
We'll definitely need it at some point in the future, better to keep.

Reply:
for me it's a loose end - maybe this whole concept should be a separate module or something